migration/migration-technical-details.md

```diff
@@ -51,12 +55,25 @@ After the pre-staking phase, a clear overview of the registered validators and t
+1. The candidate block is mined at block height 3 456 000, which opens the first activation window.
+2. The network waits for 10 confirmations to prevent block reversion.
+3. Once 10 confirmation blocks are mined, registered validators capture the state of block 3 456 000 and start generating the "genesis" block.
+4. Validators generate the "genesis" block deterministically. The block hash should be the same across all validators.
+5. After generating the "genesis" block, validators send their readiness transactions, which include the block hash in the data field.
+6. The network monitors readiness transactions within a 24 hour window.
+7. If 80% of the total stake (represented by registered validators) sends readiness transactions within 24 hours, the PoS chain starts at block height 3 456 000. If the threshold is not reached, the process automatically moves to the next activation window.
+8. If the 80% readiness threshold is not met within the first 24 hours, a new activation window begins, starting with a new candidate block at the block immediately following 1440 blocks or 24 hours from the first candidate block.
+9. In subsequent activation windows, validators regenerate the "genesis" block (3 457 440), send readiness transactions again, and the process repeats until 80% readiness is achieved.
+10. Once 80% readiness is reached, the PoS client starts, and the once candidate block becomes the transition block, marking the start of the Nimiq PoS blockchain.
 
-- **Selecting the Transition Block:** Validators target a block height corresponding to the next PoS election block as the candidate block for the transition.
-- **Executing the Transition:** If, at the candidate block's height, at least 80% of stake signals readiness for the transition, the candidate block becomes the transition block. A special activation tool captures and migrates the state of the PoW chain at the transition block, along with the transaction history, to generate the genesis block for the new PoS chain.
+#### Readiness and Activation Tool
 
-During this phase, validators send a transaction signaling their readiness. Unlike the previous two phases, it is recommended to run the activation tool before this phase begins, as migrating the entire transaction history is time-consuming. The readiness transaction is as follows:
+During the activation phase, validators send a transaction signaling their readiness. Pre-registered validators are encouraged to run the activation tool before the phase begins, as migrating the entire transaction history is time-consuming. An activation tool has been developed to facilitate this process. The tool migrates the history and scans the PoW chain for readiness transactions selecting the valid ones, which come from pre-registered validators who have completed the registration process. Additionally, the tool verifies the data field to ensure the hash generated by the client matches the actual transition block configuration hash.
```

Comment:
I did not get the last part " Additionally, the tool verifies the data field to ensure the hash generated by the client matches the actual transition block configuration hash."

Reply:
the `data` field is verified to check if the hash of the genesis is the same across validators